packages/libs/error-stack/src/fmt/hook.rs

```diff
@@ -61,8 +61,8 @@ impl HookContextImpl {
 /// ### Example
 ///
 /// ```rust
-/// # // we only test on nightly, therefore report is unused (so is render)
-/// # #![cfg_attr(not(nightly), allow(dead_code, unused_variables, unused_imports))]
+/// # // we only test on rust 1.65, therefore report is unused (so is render)
```

Comment:
```suggestion
/// # // we only test on Rust 1.65, therefore report is unused (so is render)
```

Reply:
Could you also apply this to the other locations, please?

Reply:
fixed in [7328932](https://github.com/hashintel/hash/pull/974/commits/73289325d38da6ff33218cc0fe78c6af98745922)